keras_cv/models/stable_diffusion/stable_diffusion.py

```diff
@@ -228,6 +233,216 @@ def generate_image(
         decoded = ((decoded + 1) / 2) * 255
         return np.clip(decoded, 0, 255).astype("uint8")
 
+    def text_to_image_prompt_to_prompt(
+        self,
+        prompt: str,
+        prompt_edit: str,
+        method: str,
+        self_attn_steps: Union[float, Tuple[float, float]],
```

Comment:
I'm not convinced these type annotations help readability  consider dropping and relying on docstrings

Reply:
Regarding the `prompt` I feel like that could help since there are other implementations that accept `List` of different multiple prompts as inputs for this parameter.

Additionally, there is also this issue #1368 that may, later on, alter the type passed (e.g. `Union[str, List[str, float]]`, depending on how it's decided to add prompt weighting).

But since the docstrings "hint" at the type, I am happy to change it :D

Reply:
I will close this thread since I'm removing all the type hints